pkg/api/types/volume_types.go

```diff
@@ -16,11 +16,39 @@
 
 package types
 
-import "io"
+import (
+	"context"
+	"io"
 
-type VolumeLsCommandOptions struct {
+	"github.com/containerd/containerd"
+)
+
+type VolumeCommandOptions struct {
+	GlobalCommandOptions
```

Comment:
I prefer don't use an anonymous field here. 

Maybe `GOptions *GlobalCommanOptions` would be a good choice

Reply:
Either is fine to me, but we have to keep this consistent across every commands

Reply:
This reminds me a question. If we're using `GOptions *GlobalCommanOptions` (or pointer in general for other options), we are assuming either:

1. The values in the option struct won't be changed at all, or
2. Changing values won't impact either callers or callees:
    1. If `func A` changes option values and calls `func B`, the callee B won't be impacted (e.g., B expects the changed value);
    2. If `func B` also changes option values (e.g., `B` in a separate go routinue), the caller A won't be impacted.

If we use pointers and use goroutines, we're sharing the data between different goroutines.